electron/src/update/squirrel.ts

```diff
@@ -29,20 +29,17 @@ import {getLogger} from '../logging/getLogger';
 import * as EnvironmentUtil from '../runtime/EnvironmentUtil';
 import * as lifecycle from '../runtime/lifecycle';
 import {config, MINUTE_IN_MILLIS, HOUR_IN_MILLIS} from '../settings/config';
+import {createShortcuts, removeShortcuts} from './shortcuts';
 
 const logger = getLogger(path.basename(__filename));
 
 const appFolder = path.resolve(process.execPath, '..');
 const rootFolder = path.resolve(appFolder, '..');
 const updateDotExe = path.join(rootFolder, 'Update.exe');
+const exeName = path.basename(process.execPath);
+const exePath = path.join(rootFolder, exeName);
```

Comment:
This is the main change. This is now how we compute the exec path (starting from the `root` dir of the application, not the current folder of the exec)